android/FirebasePlugin.java

```diff
@@ -304,6 +308,7 @@ public void onPause() {
   }
 
   public void onStop() {
+    stopTrace("dummy");
```

Comment:
why dummy ?

Reply:
stopTrace function have parameter. since stopTrace is invoking from js also, we can't remove that.